### DISCUSSION TITLE
Login Logfile? [fail2ban]

### DISCUSSION THREAD

**Discussion by XHyperDEVX** [YOURLS/YOURLS#3621] [Q&A]
Hello,

is there a log file that automatically stores failed logins with IP?

If no, is there another way to protect the login with fail2ban?

**Comment by dgw** [comment]
See log-login plugin at Awesome list: https://github.com/YOURLS/awesome#l

**Reply by XHyperDEVX** [reply]
yeah, thank you!!!

**Reply by XHyperDEVX** [reply]
Sorry, that doesn't fix my problem. The plugin is outdated and no longer works. At least not for me.

Is there another plugin/way?
